CMakeLists.txt

```diff
@@ -184,9 +184,13 @@ if (TTG_HAVE_CUDA OR TTG_HAVE_HIP OR TTG_HAVE_LEVEL_ZERO)
 endif()
 set(TTG_HAVE_DEVICE ${_ttg_have_device} CACHE BOOL "True if TTG has support for any device programming model")
 
-##########################
-#### prerequisite runtimes
-##########################
+############################
+#### mandatory prerequisites
+############################
+add_custom_target(ttg-imported-packages-build)
+# umpire-cxx-allocator
+include(FindOrFetchUmpireCXXAllocator)
+add_dependencies(ttg-imported-packages-build vrg-build-external-projects)
```

Comment:
Should we make this optional and only enable if we're building with CUDA/HIP/L0 support?